packages/product-json-to-csv/src/map-product-data.js

```diff
@@ -263,13 +263,17 @@ export default class ProductMapping {
 
   static _mapPriceToString(price: Price): string {
     // Full price:
-    // 'country-currencyCode centAmount|discounted.centAmount customerGroup.name#channel.key$validFrom~validUntil'
+    // 'country-currencyCode centAmount|discounted.centAmount@customerGroup.name#channel.key$validFrom~validUntil'
 
     return oneLineTrim`
       ${price.country ? `${price.country}-` : ''}
       ${price.value.currencyCode} ${price.value.centAmount}
       ${price.discounted ? `|${price.discounted.value.centAmount}` : ''}
-      ${`${price.customerGroup?.name ?? ''}`}
```

Comment:
Funny that this part was even here since we never expanded on it in the code before 🤔